app/client/src/mockResponses/ActionSettings.tsx

```diff
@@ -54,6 +54,12 @@ export const apiActionSettingsConfig = [
         controlType: "CHECKBOX",
         info: "Ask confirmation from the user each time before refreshing data",
       },
+      {
+        label: "Encode query params",
+        configProperty: "actionConfiguration.encodeParamsToggle",
+        controlType: "CHECKBOX",
+        info: "Will encode query params. Also form body for x-www-form-encoded",
```

Comment:
```suggestion
        info: "Encode query params for all APIs. Also encode form body when Content-Type header is set to x-www-form-encoded",
```

@ofpiyush Will this suggested text become too long on the screeen?